src/main/java/local/alfonso/countries/CountryComparePop.java

```diff
@@ -0,0 +1,12 @@
+package local.alfonso.countries;
+
+import java.util.Comparator;
+
+public class CountryComparePop implements Comparator<Country> {
+
+    @Override
+    public int compare(Country c1, Country c2)
+    {
+        return c1.getPopulation().compareTo(c2.getPopulation());
+    }
```

Comment:
Line 10 - since you are dealing with integers, you can just subtract `c1.getMedAge()` by `c2.getMedAge()` (doing so will produce the same result as a `String` comparison anyway.